### DISCUSSION TITLE
it/parental-control/

### DISCUSSION THREAD

**Discussion by giscus** [matteocontrini/blog#6] [Comments]
# it/parental-control/

Tra decreti Pillon e regolamenti inapplicabili, come abbiamo speso gli ultimi tre anni a ragionare su un sistema di controllo parentale che non funziona.

https://matteosonoio.it/it/parental-control/

**Comment by fcucino** [comment]
In tutto questo, confermo che la comunicazione inviata da Vodafone contiene proprio il testo "Gentile %%NOME%%"...
